ionic-team/ionic-framework
# chore(visual-regression): add script to update ground truths

Issue number: N/A

---------





## What is the current behavior?


Devs would have to manually generate the ground truths from their desired base branch. This causes a dev to checkout the base branch and pull the latest screenshots. They would then return to their working branch and start the E2E tests.

## What is the new behavior?


A script has been created to automate this process using Docker as mentioned in the design doc:

- It will ask the user a set a questions like if which component they want to test


## Does this introduce a breaking change?





## Other information



How to test:
1. Make a change to a desired component
2. Run `npm run test.e2e.script`
3. Answer the questions
4. Verify that the tests fail due to visual changes
5. Re-run the command as many times as necessary in order to try different routes based on different answers
